src/matrix-utils.js

```diff
@@ -25,12 +28,69 @@ function waitForSync(client) {
   });
 }
 
+function addScript(src) {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement("script");
+    script.setAttribute("src", src);
+    script.onload = resolve;
+    script.onerror = reject;
+    document.body.appendChild(script);
+  });
+}
+
 export async function initClient(clientOptions) {
+  await addScript(olmJsPath);
```

Comment:
Is this necessary as well as the hardcoded script addition in index.html? I think you should be able to import Olm as a module to avoid both (https://github.com/vector-im/element-web/blob/develop/src/vector/init.tsx#L87).

Reply:
Ok, I've made a change to import it that way. One gotcha I found is that `@matrix-org/olm` expects `OLM_OPTIONS` to exist in the global context. Not sure why.

Reply:
I've added an issue to the Olm repository here https://gitlab.matrix.org/matrix-org/olm/-/issues/10

Reply:
Thanks - needing OLM_OPTIONS defined is a bit weird - we don't seem to need it in element-web - not sure why though.